tsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
   "compilerOptions": {
+    "outDir": "./es",
     "noLib": false,
-    "module": "commonjs",
-    "target": "es5",
+    "module": "es6",
+    "target": "es6",
     "lib": [
```

Comment:
Can we not simply include the node_modules here instead of manually adding entries?

Reply:
that doesn't work..